ADR/ADR012-use-rails-api-for-api.md

```diff
@@ -0,0 +1,76 @@
+# ADR012: Use Rails-API for api (Replacing ruby-grape)
+
+Date: 2022-12-12
+
+## Status
+
+Proposed
```

Comment:
Think this should be `Accepted` - since we'll be accepting it by merging it in

```suggestion
Accepted
```

Reply:
Fixed